### ISSUE TITLE
swc support

### ISSUE THREAD

**Issue by oliviertassinari** [oliviertassinari/babel-plugin-transform-react-remove-prop-types#201]
I wonder if we shouldn't port this plugin to https://swc.rs/docs/usage/plugins. It could unlock the support of https://nextjs.org/docs/messages/swc-disabled.

**Comment by Sandstedt** [comment]
Yes that would be great. We have still not converted to typescript, so we use prop-types heavily in our projects. Let me know if you need any help!

**Comment by mickaelalvs** [comment]
👋🏼 Is there a solution for doing the same thing with swc now? 😊

**Comment by askoufis** [comment]
> 👋🏼 Is there a solution for doing the same thing with swc now? 😊

There is https://www.npmjs.com/package/@modern-js/swc-plugins, however the prop-types removal plugin is not published as its own plugin, but rather as part of a `Compiler` API. There's [an issue](https://github.com/web-infra-dev/swc-plugins/issues/214) related to publishing standalone plugins (which you seem to have found already @CruuzAzul).

**Comment by mickaelalvs** [comment]
Thanks @askoufis ! 😊 In fact I've already read about the US on their repo and I've even added one with my problem 😅 https://github.com/web-infra-dev/swc-plugins/issues/215

**Comment by oliviertassinari** [comment]
Considering where swc is going, it's actually oxc that should be invested in now: https://oxc.rs/.
